Wiktionary_scraper.py

```diff
@@ -91,9 +95,15 @@ def read_csv(filename):
 
 
 def read_wordlist(filename):
-    file = open(filename, 'r')
+    file = open(filename, 'r', encoding="utf-8")
```

Comment:
`with open` it's safer. in this case you never close the file you are opening. it's not a good way to do.

Reply:
also do not use `file` as variable name. use something like: `fp` is the common name for those.